storybook/mocks/wallet.ts

```diff
@@ -234,18 +234,18 @@ export const mockYoroiSignedTx: YoroiSignedTx & {mock: true} = {
   mock: true,
 }
 
-export const mockKeyStore = (overrides?: {
-  getData?: typeof KeyStore.getData
-  storeData?: typeof KeyStore.storeData
-  deleteData?: typeof KeyStore.deleteData
+export const mockMasterKey = (overrides?: {
+  reveal?: ReturnType<MasterKey>['reveal']
+  keep?: ReturnType<MasterKey>['keep']
+  discard?: ReturnType<MasterKey>['discard']
 }) =>
   ({
-    getData: async (_keyId, _encrpytionMethod, _message, password, _intl) => {
+    reveal: async (password) => {
       if (password !== 'password') throw new Error('Invalid Password')
 
       return 'masterkey'
     },
-    storeData: async () => undefined,
-    deleteData: async () => undefined,
+    keep: async () => undefined,
```

Comment:
`throw new Error("Not implemented")`?

Reply:
hmm, I think if someone wants another behavior then the default"ed" mock, it should pass it thought `overrides`, when using this mock I'd expect to have these methods mocked doing nothing.